src/pages/solutions/spatial-data-management.astro

```diff
@@ -80,12 +82,12 @@ const page: IStandardPageMetadata = {
       class="w-full max-w-[400px]"
     />
     <p>
-      Government runs better with a shared vision and shared data available to all for decision making. Good data is
-      timely, accurate and, most importantly, used. Getting your data into the hands of the people that need it on time
-      and in the right format is important, and may require the ability to maintain data in multiple locations for
-      different purposes. Additionally, understanding the uses of your data critically influences the update cycle to
-      keep it current and reliable. Our long history of data wrangling has taught us a lot about data accessibility and
-      currency that we can pass along and enable for your team, program, or agency.
+      Democracy depends on distributing accurate information as widely as possible to help people make informed
```

Comment:
I can't put my finger on it, but there's something about this sentence that I want to suggest a change. Maybe it feels like "democracy" evokes too much of a political vibe for our website. What about "Efficient and effective government", or something like that? Also, in this sentence we talk about distributing information, then in the next it's about making data available. Should we say "distributing accurate data as widely as possible to help people make informed decisions" since the data _becomes_ the information?

Reply:
Yeah, I switched to "democracy" because I thought it felt too "weasel wordy" originally, but the more I look at it the more I think you're right. Good catch on the information/data stuff.